<ui version="4.0" >
 <class>generalPreferencesWidget</class>
 <widget class="QWidget" name="generalPreferencesWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="preferencesGroupBox" >
     <property name="title" >
      <string>Preferences</string>
     </property>
     <layout class="QGridLayout" name="preferencesGridLayout" >
      <property name="horizontalSpacing" >
       <number>12</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="loglevelLabel" >
        <property name="text" >
         <string>Log level</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <layout class="QHBoxLayout" name="loglevelHorizontalLayout" >
        <item>
         <widget class="QComboBox" name="loglevelComboBox" >
          <item>
           <property name="text" >
            <string>DEBUG</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>INFO</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>WARNING</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>ERROR</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>CRITICAL</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="loglevelHorizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>138</width>
            <height>24</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="cachedirLabel" >
        <property name="text" >
         <string>Cache directory location</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="FileEntryWidget" native="1" name="cachedirEntryWidget" >
        <property name="toolTip" >
         <string>Location of the directory for file cache.
Default: "$HOME/.gsdview".</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filedialogGroupBox" >
     <property name="title" >
      <string>File DIalog</string>
     </property>
     <layout class="QVBoxLayout" name="filedialogVerticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="workdirHorizontalLayout" >
        <property name="spacing" >
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="workdirLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Working directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="FileEntryWidget" native="1" name="workdirEntryWidget" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement" >
           <size>
            <width>1</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Base directory for files and directories selection.
Default: "$HOME".</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="foldersGroupBox" >
        <property name="title" >
         <string>Favorite Folders</string>
        </property>
        <layout class="QVBoxLayout" name="foldersVerticalLayout" >
         <item>
          <widget class="QListWidget" name="foldersListWidget" >
           <property name="contextMenuPolicy" >
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="acceptDrops" >
            <bool>true</bool>
           </property>
           <property name="toolTip" >
            <string>Favorite folders are showed in the left sidebar of the file chooser dialog.</string>
           </property>
           <property name="selectionMode" >
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="folderButtonsHorizontalLayout" >
           <item>
            <spacer name="fodersButtonsHorizontalSpacer" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addFolderButton" >
             <property name="toolTip" >
              <string>Ad a new item</string>
             </property>
             <property name="text" >
              <string>Add</string>
             </property>
             <property name="icon" >
              <iconset resource="../../resources.qrc" >
               <normaloff>:/images/add.svg</normaloff>:/images/add.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editFolderButton" >
             <property name="toolTip" >
              <string>Edit selected item</string>
             </property>
             <property name="text" >
              <string>Edit</string>
             </property>
             <property name="icon" >
              <iconset resource="../../resources.qrc" >
               <normaloff>:/images/edit.svg</normaloff>:/images/edit.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeFolderButton" >
             <property name="toolTip" >
              <string>Remove selected items</string>
             </property>
             <property name="text" >
              <string>Remove</string>
             </property>
             <property name="icon" >
              <iconset resource="../../resources.qrc" >
               <normaloff>:/images/remove.svg</normaloff>:/images/remove.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearFoldersButton" >
             <property name="toolTip" >
              <string>Clear all items</string>
             </property>
             <property name="text" >
              <string>Clear</string>
             </property>
             <property name="icon" >
              <iconset resource="../../resources.qrc" >
               <normaloff>:/images/clear.svg</normaloff>:/images/clear.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionAddFavorite" >
   <property name="icon" >
    <iconset resource="../../resources.qrc" >
     <normaloff>:/images/add.svg</normaloff>:/images/add.svg</iconset>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
  </action>
  <action name="actionRemoveFavorite" >
   <property name="icon" >
    <iconset resource="../../resources.qrc" >
     <normaloff>:/images/remove.svg</normaloff>:/images/remove.svg</iconset>
   </property>
   <property name="text" >
    <string>Remove</string>
   </property>
  </action>
  <action name="actionEditFavorite" >
   <property name="icon" >
    <iconset resource="../../resources.qrc" >
     <normaloff>:/images/edit.svg</normaloff>:/images/edit.svg</iconset>
   </property>
   <property name="text" >
    <string>Edit</string>
   </property>
  </action>
  <action name="actionClearFavorites" >
   <property name="icon" >
    <iconset resource="../../resources.qrc" >
     <normaloff>:/images/clear.svg</normaloff>:/images/clear.svg</iconset>
   </property>
   <property name="text" >
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileEntryWidget</class>
   <extends>QWidget</extends>
   <header>widgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>addFolderButton</sender>
   <signal>clicked()</signal>
   <receiver>actionAddFavorite</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>313</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editFolderButton</sender>
   <signal>clicked()</signal>
   <receiver>actionEditFavorite</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>394</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeFolderButton</sender>
   <signal>clicked()</signal>
   <receiver>actionRemoveFavorite</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>477</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearFoldersButton</sender>
   <signal>clicked()</signal>
   <receiver>actionClearFavorites</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>558</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
